pkg/playbook/app/vcertutil/helper.go

```diff
@@ -142,9 +152,15 @@ func setLocationWorkload(playbookRequest domain.PlaybookRequest, vcertRequest *c
 	segments := strings.Split(playbookRequest.Location.Instance, ":")
 	instance := segments[0]
 	workload := ""
+	// take workload from instance string
 	if len(segments) > 1 {
 		workload = segments[1]
 	}
+	// take workload from attribute.
+	// workload attribute has priority over workload string declared in request.Location.Instance
+	if playbookRequest.Location.Workload != "" {
```

Comment:
This is not in the README for Playbook, would you mind adding it for this changes?
### Location

| Field      | Type    | Description |
|------------|---------|-------------|
| instance   | string  ||
| tlsAddress | string  ||
| replace    | boolean ||

->

### Location

| Field      | Type    | Description |
|------------|---------|-------------|
| instance   | string  ||
| tlsAddress | string  ||
| workload | string  ||
| replace    | boolean ||

Reply:
this will be added by UA